<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_windowClass</class>
 <widget class="QMainWindow" name="main_windowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_window</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionImportFiles"/>
    <addaction name="actionSaveFiles"/>
   </widget>
   <widget class="QMenu" name="menuPaint">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSet_Visible"/>
    <addaction name="actionSet_Invisible"/>
    <addaction name="actionObject_Color"/>
    <addaction name="actionVertex_Color"/>
    <addaction name="actionLabel_Color"/>
    <addaction name="actionOriginal_Location"/>
    <addaction name="actionShow_Tracjectory"/>
    <addaction name="actionDont_Trace"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="actionScene_Mode"/>
    <addaction name="actionSelect_Mode"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithm">
    <property name="title">
     <string>Algorithm</string>
    </property>
    <addaction name="actionClustering"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionScanner"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuPaint"/>
   <addaction name="menuSelect"/>
   <addaction name="menuAlgorithm"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImportFiles"/>
   <addaction name="actionSaveFiles"/>
   <addaction name="actionScanner"/>
   <addaction name="separator"/>
   <addaction name="actionSet_Visible"/>
   <addaction name="actionSet_Invisible"/>
   <addaction name="actionShow_Tracjectory"/>
   <addaction name="actionDont_Trace"/>
   <addaction name="separator"/>
   <addaction name="actionObject_Color"/>
   <addaction name="actionVertex_Color"/>
   <addaction name="actionLabel_Color"/>
   <addaction name="actionOriginal_Location"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Mode"/>
   <addaction name="actionScene_Mode"/>
   <addaction name="actionClustering"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>38</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>549</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <action name="actionImportFiles">
   <property name="icon">
    <iconset>
     <normaloff>Resources/openFile.png</normaloff>Resources/openFile.png</iconset>
   </property>
   <property name="text">
    <string>ImportFiles</string>
   </property>
  </action>
  <action name="actionSet_Visible">
   <property name="icon">
    <iconset>
     <normaloff>Resources/visible.png</normaloff>Resources/visible.png</iconset>
   </property>
   <property name="text">
    <string>Set Visible</string>
   </property>
  </action>
  <action name="actionSet_Invisible">
   <property name="icon">
    <iconset>
     <normaloff>Resources/invisible.png</normaloff>Resources/invisible.png</iconset>
   </property>
   <property name="text">
    <string>Set Invisible</string>
   </property>
  </action>
  <action name="actionScene_Mode">
   <property name="icon">
    <iconset>
     <normaloff>Resources/scene.png</normaloff>Resources/scene.png</iconset>
   </property>
   <property name="text">
    <string>Scene Mode</string>
   </property>
  </action>
  <action name="actionSelect_Mode">
   <property name="icon">
    <iconset>
     <normaloff>Resources/select.png</normaloff>Resources/select.png</iconset>
   </property>
   <property name="text">
    <string>Select Mode</string>
   </property>
  </action>
  <action name="actionClustering">
   <property name="icon">
    <iconset>
     <normaloff>Resources/categorize.png</normaloff>Resources/categorize.png</iconset>
   </property>
   <property name="text">
    <string>Clustering</string>
   </property>
  </action>
  <action name="actionObject_Color">
   <property name="icon">
    <iconset>
     <normaloff>Resources/tree.png</normaloff>Resources/tree.png</iconset>
   </property>
   <property name="text">
    <string>Object Color</string>
   </property>
  </action>
  <action name="actionVertex_Color">
   <property name="icon">
    <iconset>
     <normaloff>Resources/leaf.png</normaloff>Resources/leaf.png</iconset>
   </property>
   <property name="text">
    <string>Vertex Color</string>
   </property>
  </action>
  <action name="actionLabel_Color">
   <property name="icon">
    <iconset>
     <normaloff>Resources/label.png</normaloff>Resources/label.png</iconset>
   </property>
   <property name="text">
    <string>Label Color</string>
   </property>
  </action>
  <action name="actionOriginal_Location">
   <property name="icon">
    <iconset>
     <normaloff>Resources/tree2.png</normaloff>Resources/tree2.png</iconset>
   </property>
   <property name="text">
    <string>Original Location</string>
   </property>
  </action>
  <action name="actionShow_Tracjectory">
   <property name="icon">
    <iconset>
     <normaloff>Resources/show_trace.png</normaloff>Resources/show_trace.png</iconset>
   </property>
   <property name="text">
    <string>Show Tracjectory</string>
   </property>
  </action>
  <action name="actionDont_Trace">
   <property name="icon">
    <iconset>
     <normaloff>Resources/dont_trace.png</normaloff>Resources/dont_trace.png</iconset>
   </property>
   <property name="text">
    <string>Dont Trace</string>
   </property>
  </action>
  <action name="actionScanner">
   <property name="icon">
    <iconset>
     <normaloff>Resources/camera.png</normaloff>Resources/camera.png</iconset>
   </property>
   <property name="text">
    <string>Scanner</string>
   </property>
  </action>
  <action name="actionSaveFiles">
   <property name="icon">
    <iconset>
     <normaloff>Resources/save_file.png</normaloff>Resources/save_file.png</iconset>
   </property>
   <property name="text">
    <string>saveFiles</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="main_window.qrc"/>
 </resources>
 <connections/>
</ui>
